src/main/java/org/folio/oaipmh/Constants.java

```diff
@@ -92,4 +94,9 @@ private Constants() {
   public static final String MATERIAL_TYPES = "materialType";
   public static final String INSTANCE_TYPES = "resourceType";
   public static final String INSTANCE_FORMATS = "format";
+  public static final String RETRY_ATTEMPTS = "retryAttempts";
+  public static final String STATUS_CODE = "statusCode";
+  public static final String STATUS_MESSAGE = "statusMessage";
+
+  public static final Integer RESUMPTION_TOKEN_TIMEOUT = 600;
```

Comment:
Resumption token expires after 24 hours since the last request to oai-pmh, so make the value to be equal to 24h.